### ISSUE TITLE
all-in-one with deauth

### ISSUE THREAD

**Issue by SiNaPsEr0x** [bigbrodude6119/flipper-zero-evil-portal#84]
Hello ... it would be nice now with the latest upgrade to integrate [bgloader](https://github.com/twisted-pear/bgloader) between evil portal and the deauth .. in this way there is no going crazy with wpa keys as by sending down the network, he would hire the same ssid and with a fake portal a user could directly enter his credentials ... such a job was done with the fluxion tool (present on github)

Do you think it's possible to do it?

**Comment by Spessmen14** [comment]
I don't think this is possible without two esp-32 boards, because you need one for the WiFi running the portal and another to deauth the targeted AP. Splitting a single physical interface into 'half' to do both tasks is only possible with a very few wifi chipsets (read up on the ones compatible with Kali linux) and the esp-32's wifi isn't one of them.
